api/src/schemas.py

```diff
@@ -60,7 +60,9 @@ class DashboardCollection(Collection):
     item_assets: Optional[Dict]
```

Comment:
I'm not sure we want to make this change for VEDA - we want users to explicitly include these fields, and it would make more sense for us to use something like this:

```
is_periodic: bool = Field(..., alias="dashboard:is_periodic")
time_density: Optional[str] = Field(..., alias="dashboard:time_density")
```

For MAAP's use case, it might be better to use the `Collection` model rather than VEDA's `DashboardCollection`.

Reply:
> we often want users to explicitly include these fields

I see what you mean. That said

> For MAAP's use case, it might be better to use the Collection model rather than VEDA's DashboardCollection.

The problem I think is that we don't have a MAAP fork of the `veda-stac-ingestor` -- and I don't think we want to have one, it would be duplicating maintenance time.

Reply:
it sounds like these being optional is the goal of this issue : https://github.com/NASA-IMPACT/veda-stac-ingestor/issues/53.

Reply:
yeah, they should be optional so we can publish any valid STAC record to the catalog